.gitignore

```diff
@@ -14,3 +14,7 @@ buildSrc/build
 .externalNativeBuild
 .cxx
 local.properties
+gradlew
```

Comment:
In my personal experience, it's good to submit this file in the git repository. In the future, if you want to integrate a Continuous Integration to validate and test your application, you'll probably need this file to run o Linux/Mac VMs.